sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ActiveClientTokenManager.java

```diff
@@ -85,7 +80,11 @@ public ActiveClientTokenManager(Mono<ClaimsBasedSecurityNode> cbsNode, String to
 
                     final Duration firstInterval = Duration.ofMillis(refreshIntervalMS);
                     lastRefreshInterval.set(firstInterval);
-                    authorizationResultsSink.next(AmqpResponseCode.ACCEPTED);
+                    authorizationResults.emitNext(AmqpResponseCode.ACCEPTED, (signalType, emitResult) -> {
+                        logger.info("signalType[{}] result[{}] Could not emit ACCEPTED.", signalType, emitResult);
```

Comment:
How important are these emission, completion failures to the overall application? Is it ignorable or could it lead to downstream issues? What I'm wondering is if these failures should be logged at `info` or `verbose` depending on how often they occur and their importance.

Reply:
Good call.. I have been trying to either remove or lower levels because some of them are rather verbose.

Reply:
Yeah, making this verbose would be better as the user really can't diagnose much from this as it's mostly an implementation detail.